aten/src/ATen/native/native_functions.yaml

```diff
@@ -1010,7 +1010,7 @@
   use_c10_dispatcher: full
   variants: function, method
   dispatch:
-    CPU: legacy::cpu::_th_dot
+    CPU: dot
     CUDA: legacy::cuda::_th_dot
 
 - func: dot.out(Tensor self, Tensor tensor, *, Tensor(a!) out) -> Tensor(a!)
```

Comment:
I wonder what's going on with dot.out overload - there is no implementation?

Reply:
There is one at 

https://github.com/pytorch/pytorch/blob/0309f6a4bb2b739adcec6bc36eb151d7ddb27e14/aten/src/ATen/native/LinearAlgebra.cpp#L367-L372

Reply:
yeah, found it. Must be one of the few cases where `_out` version calls a regular one, not vice versa :-)